src/raven.js

```diff
@@ -498,6 +496,41 @@ Raven.prototype = {
         }
     },
 
+    showReportDialog: function (options) {
+        if (!window.document) // doesn't work without a document (React native)
+            return;
+
+        options = options || {};
+
+        var lastEventId = options.eventId || this.lastEventId();
+        if (!lastEventId) {
+            throw new RavenConfigError('Missing eventId');
```

Comment:
We probably shouldn't throw errors like this since they'd then get caught and in turn reported to Sentry. Thoughts?

Or we have a rule to ignore RavenConfigError's. This wasn't an issue before because RavenConfigError was only ever thrown before raven was even hooked up. Now we'll be throwing errors that will be caught and logged.

Not sure if bad or not, just pointing it out.